esp/src/src-react/components/Helpers.tsx

```diff
@@ -100,6 +101,9 @@ export const Helpers: React.FunctionComponent<HelpersProps> = ({
     const [treeItems, setTreeItems] = React.useState<FlatTreeItemEx[]>([]);
     const [openItems, setOpenItems] = React.useState<Iterable<TreeItemValue>>([]);
     const [selectedTreeItem, setSelectedTreeItem] = React.useState<FlatTreeItemEx>();
+    const [showThorSlaveLogs, setShowThorSlaveLogs] = React.useState(false);
+
+    const { workunit } = useWorkunit(wuid, true);
```

Comment:
It would be better to add some kind of boolean prop to Helpers, something like thorLogsAvailable(?), which you would assign to <Helpers... > in WorkunitDetails.tsx by looking at its instance of workunit?.ThorLogList. 

I say that because every use of useWorkunit() results in a network request for data that's often already been fetched.